docs/operation/volume_management/volume_creation_deletion_cli.rst

```diff
@@ -26,13 +26,16 @@ This section describes how to create a **Volume** from the **CLI**.
          storageClassName: <storageclass_name>
          rawBlockDevice:
            devicePath: <device_path>
+           noFormat: false
 
    Set the following fields:
 
    - **name**: the name of your volume, must be unique
    - **nodeName**: the name of the node where the volume will be located.
    - **storageClassName**: the **StorageClass** to use
    - **devicePath**: path to the block device (for example, `/dev/sda1`).
+   - **noFormat**: skip the volume formatting if **true**, optional (default to
```

Comment:
Should `noFormat` be inside the `rawBlockDevice` section? Why not top-level?

Also, instead of `noFormat`, could we re-use the `volumeMode` as used `PersistentVolume(Claim)`?

Reply:
Yeah, good point.

I'll add a commit to change that.